drivers/amazonec2/amazonec2.go

```diff
@@ -566,12 +573,21 @@ func (d *Driver) Create() error {
 			DeleteOnTermination: aws.Bool(true),
 		},
 	}
-	netSpecs := []*ec2.InstanceNetworkInterfaceSpecification{{
-		DeviceIndex:              aws.Int64(0), // eth0
-		Groups:                   makePointerSlice(d.securityGroupIds()),
-		SubnetId:                 &d.SubnetId,
-		AssociatePublicIpAddress: aws.Bool(!d.PrivateIPOnly),
-	}}
+
+	var netSpecs []*ec2.InstanceNetworkInterfaceSpecification = nil
+	var securityGroupIds []*string = nil
```

Comment:
Why are these `*string` instead of `string`?  Seems a potential nil deference panic